File: Leet Code/Java/Problem69.java
import java.util.Scanner;

public class Problem69 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        System.out.print("Enter number: ");
        int num = in.nextInt();
        int sqRoot = (int) Math.sqrt(num);
        System.out.println("Square root of " + num + " is " + sqRoot);
    }
}

File: Leet Code/Python/problem_69.py

import math

class Solution(object):
    def mySqrt(self, x):
        sqRoot = int(math.sqrt(x))
        return sqRoot

num = int(input("Enter number: "))
sq = Solution()
print(f'Square root of {num} is {sq.mySqrt(num)}')
